Marsiple/Common/UI/TitleAndBodyView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  PostView.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 20/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import UIKit
+
+class TitleAndBodyView: UIView {
+    private let titleLabel = UILabel.autolayoutView()
+    private let bodyLabel = UILabel.autolayoutView()
+    var titleLabelFontSize: CGFloat = 20 {
```

Comment:
Lets add a newline here.

Reply:
`titleLabelFontSize`and `bodyLabelFontSize` are exposed. Should they be above private properties?

Reply:
Yep, lets put them above.